Gemfile

```diff
@@ -17,3 +17,5 @@ gem "folio", github: "sinfin/folio", branch: "master"
 
 # To use a debugger
 # gem 'byebug', group: [:development, :test]
+
+gem "rails-i18n", "~> 6"
```

Comment:
Tohle je nutné? Jsem myslel, že to v sobě mají už Rails.

Reply:
Automaticky se to zvedlo na verzi 7 a rozbilo to cele auctify.